tests/pykafka/test_protocol.py

```diff
@@ -256,5 +256,95 @@ def test_offset_fetch_response(self):
         self.assertEqual(response.topics[b'emmett.dummy'][0].offset, 1)
 
 
+class TestGroupMembershipAPI(unittest2.TestCase):
+    maxDiff = None
+
+    def test_consumer_group_protocol_metadata(self):
+        meta = protocol.ConsumerGroupProtocolMetadata()
+        msg = meta.get_bytes()
+        self.assertEqual(
+            msg,
+            bytearray(b'\x00\x00\x00\x00\x00\x01\x00\ndummytopic\x00\x00\x00\x0ctestuserdata')
```

Comment:
For this one as well, I'm proposing what is a simple testing utility that will improve the tests being added by this pull request. It's a judgement call, but I'll advocate that we look to improve utilities on every opportunity instead of pushing them into some future.